editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -2416,6 +2417,53 @@ export function ProjectContentTreeRootKeepDeepEquality(): KeepDeepEqualityCall<P
   return objectDeepEquality(ProjectContentsTreeKeepDeepEquality())
 }
 
+export const NullableGithubChecksumsKeepDeepEquality: KeepDeepEqualityCall<GithubChecksums | null> =
+  createCallWithTripleEquals<GithubChecksums | null>()
+
+const GithubChecksumsKeepDeepEquality: KeepDeepEqualityCall<GithubChecksums | null> = (
+  oldAttribute,
+  newAttribute,
+) => {
+  if (oldAttribute == null && newAttribute == null) {
+    return keepDeepEqualityResult(oldAttribute, true)
+  }
+  if (oldAttribute == null) {
+    return keepDeepEqualityResult(newAttribute, false)
+  }
+  if (newAttribute == null) {
+    return keepDeepEqualityResult(oldAttribute, false)
+  }
+  return objectDeepEquality(StringKeepDeepEquality)(oldAttribute, newAttribute)
+}
+
+const GithubFileChangesKeepDeepEquality: KeepDeepEqualityCall<GithubFileChanges | null> = (
```

Comment:
I think this is now unused! if so, to the bin with it!

Reply:
oh, good catch

Reply:
https://github.com/concrete-utopia/utopia/pull/2733/commits/c4c4672691ab949ef4917343733281da9baf95e4 (and https://github.com/concrete-utopia/utopia/pull/2733/commits/bfdc39cfcdb9e0436673ec3f6f34ea6df387dad8 which moves the selectors and `GithubFileChanges` to the GH helpers file)